portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerModule.java

```diff
@@ -62,7 +62,7 @@ private static TransferExtension findTransferExtension(
     try {
       return transferExtensions
           .stream()
-          .filter(ext -> ext.getServiceId().equals(service))
+          .filter(ext -> ext.getServiceId().equals(service.toLowerCase()))
```

Comment:
Can we write a test that ensures a mixed or lower case service name works here?

Reply:
Good point, done. And it caught a bug :)